KartingCoach/Model/LapTime.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  LapTime.swift
+//  KartingCoach
+//
+//  Created by Jakub Olejník on 01/09/2017.
+//
+
+import Foundation
+
+struct LapTime {
+    static var zero: LapTime { return LapTime(duration: 0) }
+    
+    let minutes: UInt
+    let seconds: UInt
+    let miliseconds: UInt
+    
+    var duration: UInt {
+        return miliseconds + seconds * 1000 + minutes * 60 * 1000
+    }
+}
+
+extension LapTime {
+    /**
+      * Initialize new lap time
+      *
+      * - parameter duration: Duration of lap time in miliseconds
+      */
+    init(duration: UInt) {
+        self.init(minutes: duration / 1000 / 60, seconds: duration / 1000 % 60, miliseconds: duration % 1000)
+    }
+}
+
+extension LapTime: CustomStringConvertible {
+    var description: String {
+        let decimalSeparator = Locale.current.decimalSeparator ?? "."
+        let secondsAndMiliseconds = String(seconds) + decimalSeparator + String(miliseconds)
+        return minutes > 0 ? String(minutes) + ":" + secondsAndMiliseconds : secondsAndMiliseconds
+    }
+}
+
+extension LapTime: CustomDebugStringConvertible {
+    var debugDescription: String {
+        return "\(self)"
+    }
+}
+
+extension LapTime: Comparable {
+    static func<(lhs: LapTime, rhs: LapTime) -> Bool {
+        return lhs.duration < rhs.duration
+    }
+    
+    static func==(lhs: LapTime, rhs: LapTime) -> Bool {
```

Comment:
Prozenu ti to potom swiftlintem, operatory maji mit mezeru z obou stran 😉

Reply:
No z obou stran pri volani ne? Pri definici, mozna zleva, ale z obou se mi to nelibi :-D

Reply:
https://github.com/github/swift-style-guide#use-whitespace-around-operator-definitions 😉

Reply:
No jo no...